deploy/charts/cert-manager/BUILD.bazel

```diff
@@ -23,50 +54,16 @@ helm_pkg(
     tpl_files = [
         "//deploy/charts/cert-manager/templates:_helpers.tpl",
     ],
-    values_yaml = ":values.yaml",
-)
-
-genrule(
-    name = "README",
-    srcs = [
-        "README.template.md",
-        "//:version",
-    ],
-    outs = ["README.md"],
-    cmd = "cat $(location README.template.md) | sed -e \"s:{{RELEASE_VERSION}}:$$(cat $(location //:version)):g\" > $@",
-    stamp = 1,
-    visibility = ["//visibility:public"],
-)
-
-# We don't want alpha and beta releases to show on ArtifactHUB as releases.
-# To prevent pre-releases from appearing, we use the ArtifactHUB-specific:
-#
-#   annotation:
-#     artifacthub.io/prerelease: "true"
-#
-# See https://artifacthub.io/docs/topics/annotations/helm/
```

Comment:
This helpful comment should be moved to the Chart.template.yaml ...maybe shortened a little.

Reply:
I haven't finished all the documentation in the README.md file yet, just was looking for some feedback on the method etc. Once we agree on that, I'll make sure the docs are correct.

Reply:
OK